PERFTEST.PS1

```diff
@@ -5,7 +5,6 @@ Param(
     [parameter(Mandatory=$true)]  [string] $SrcIp, 
     [parameter(Mandatory=$true)]  [string] $DestIp,
     [parameter(Mandatory=$true)]  [ValidateScript({Test-Path $_ -PathType Container})] [String] $OutDir = "",
-    [ValidateSet("Default", "Azure", "Detail", "Max", "Container")]
```

Comment:
Did you mean to remove this line?

Reply:
Yes, this prevents adding new configs for one off tests.